### DISCUSSION TITLE
how to Query Date Range in filter with Http Request

### DISCUSSION THREAD

**Discussion by tonyhart7** [pocketbase/pocketbase#2435] [Q&A]
![image](https://user-images.githubusercontent.com/42036827/236660542-3d6f1844-6de0-4f34-82c1-c21651edaa51.png)

like from user id = 10 and (date >= start time and date <= end time)

**Comment by ganigeorgiev** [comment]
The above condition with `>=` and `<=` should work but it seems that the filter expression in the above screenshot is missing `&&` after `direver_id='test'`, aka. you can try with:

```js
driver_id='test' && date >= '2023-04-28 00:00:00.000Z' && date <= '2023-05-10 00:00:00.000Z'
```

**Reply by tonyhart7** [reply]
postman auto replace && with %26%26 and make it completely unreadable for human in few var

**Reply by ganigeorgiev** [reply]
The url encoding is fine but `&&` (or `%26%26`) is missing between the `driver_id='test'` and before the `(date...` expression.

**Reply by tonyhart7** [reply]
![image](https://user-images.githubusercontent.com/42036827/236663394-ce17aaf9-3954-4950-a9a9-db58e78df8dd.png)

I use (..) from doc, its works but idk what the trade off or side effect

**Reply by ganigeorgiev** [reply]
I'm not sure that I follow. There is nothing wrong in enclosing the expression with brackets/parentheses and they could be used for grouping conditions (eg. `field1 = true && (field2 = false || field3 = true)`).

In your case the brackets/parentheses are not necessary since all 3 conditions are combined with `&&` (AND) and the end result would be the same.
